src/main/java/org/folio/models/FundCodeExpenseClassesHolder.java

```diff
@@ -0,0 +1,170 @@
+      ledgerIdVsLedgerMap.put(ledger.getId(), ledger);
+    }
+    List<Fund> fundList = getFundList();
+    for (Ledger ledger : ledgerList) {
+      for (Fund fund : fundList) {
+        if (Objects.equals(ledger.getId(), fund.getLedgerId())) {
+          FundCodeVsExpClassesType fundCodeVsExpenseClassesType = new FundCodeVsExpClassesType();
+          fundCodeVsExpenseClassesType.setFundCode(fund.getCode());
+          fundCodeVsExpenseClassesType.setLedgerCode(ledgerIdVsLedgerMap.get(fund.getLedgerId()).getCode());
+          fundCodeVsExpenseClassesType.setActiveFundCodeVsExpClasses(getActiveStatusBudgetExpenseClass(fund));
+          fundCodeVsExpenseClassesType.setInactiveFundCodeVsExpClasses(getInActiveStatusBudgetExpenseClass(fund));
+          fundCodeVsExpenseClassesTypeList.add(fundCodeVsExpenseClassesType);
+        }
+      }
+    }
+    return getFundCodeVsExpenseClassesTypeCollection(fundCodeVsExpenseClassesTypeList);
+  }
+
+  private List<String> getActiveStatusBudgetExpenseClass(Fund fund) {
+    List<BudgetExpenseClass> budgetExpenseClassList = getBudgetExpenseClassList();
+    List<Budget> budgetList = getBudgetCollection().getBudgets();
+    List<String> activeStatus = new ArrayList<>();
+    List<Budget> budgetListByFundId = new ArrayList<>();
+    for (Budget budget : budgetList) {
+      if (Objects.equals(budget.getFundId(), fund.getId())) {
+        budgetListByFundId.add(budget);
+      }
+    }
+    for (Budget budget : budgetListByFundId) {
+      for (BudgetExpenseClass budgetExpenseClass : budgetExpenseClassList) {
+        if (Objects.equals(budget.getId(), budgetExpenseClass.getBudgetId())) {
+          if (budgetExpenseClass.getStatus().equals(BudgetExpenseClass.Status.fromValue("Active"))) {
+            activeStatus.add(addFundCodeAndExpanseClassCode(budgetExpenseClass, fund));
+          }
+        }
+      }
+    }
+    return activeStatus;
+  }
+
+  private String addFundCodeAndExpanseClassCode(BudgetExpenseClass budgetExpenseClass, Fund fund) {
+    for (ExpenseClass expenseClass : getExpenseClassList()) {
+      if (Objects.equals(budgetExpenseClass.getExpenseClassId(), expenseClass.getId())) {
+        return fund.getCode() + ":" + expenseClass.getCode();
+      }
+    }
+    return null;
+  }
+
+  private List<String> getInActiveStatusBudgetExpenseClass(Fund fund) {
+    List<BudgetExpenseClass> budgetExpenseClassList = getBudgetExpenseClassList();
+    List<Budget> budgetList = getBudgetCollection().getBudgets();
+    List<String> inActiveStatus = new ArrayList<>();
+    List<Budget> budgetListByFundId = new ArrayList<>();
+    for (Budget budget : budgetList) {
+      if (Objects.equals(budget.getFundId(), fund.getId())) {
+        budgetListByFundId.add(budget);
+      }
+    }
+    for (Budget budget : budgetListByFundId) {
+      for (BudgetExpenseClass budgetExpenseClass : budgetExpenseClassList) {
+        if (Objects.equals(budget.getId(), budgetExpenseClass.getBudgetId())) {
```

Comment:
Both values are of `String` type. You can simplify the expression using `String.equals()` method